packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -74,6 +74,11 @@ export interface IDropdownProps extends React.Props<Dropdown> {
    */
   disabled?: boolean;
 
+  /**
+   * Whether or not the Dropdown is required.
+   */

```

Comment:
Can you run `rush change` and require a minor fix with the comment:

"Dropdown: Adding a required property which will be passed into the Label."

This is how auto publishing knows what you want to be included in the published build.